pkg/connectors/openapiclient/.travis.yml

```diff
@@ -0,0 +1,8 @@
+language: go
```

Comment:
this file can be deleted from the makefile as we dont use it. the same way we are deleting `go.mod `and `go.sum `in the generated client code.

Reply:
Deleted